test/ContentClient.spec.ts

```diff
@@ -308,40 +308,12 @@ describe('ContentClient', () => {
     expect(result).toEqual([deployment])
   })
 
-  it('When fetching all deployments with offset pagination, then the result is as expected', async () => {
+  it('When fetching all deployments with pagination, then subsequent calls are made correctly', async () => {
```

Comment:
how are you testing that subsequent calls are made correctly?

Reply:
Lol, good point. Just changed it